js/scripts.js

```diff
@@ -0,0 +1,20 @@
+
+
+function getTriangleArea(a, h) {
+    
+    if (a, h <= 0) {
+        alert('nieprawilowe dane');
+    }
+    else {
+        return (a * h / 2);
+    }
+}
+console.log(getTriangleArea(10,6)); 
+
+var triangle1Area = getTriangleArea(10,15);
+var triangle2Area = getTriangleArea(3,15);
+var triangle3Area = getTriangleArea(12,5);
+
+console.log (triangle1Area);
```

Comment:
`log` to normalna funkcja/metoda - nie dajemy przed jej parametrami spacji, tak samo jak nie dałeś w `getTriangleArea(10,15)`